library/src/main/java/org/xmtp/android/library/frames/OpenFramesProxy.kt

```diff
@@ -0,0 +1,26 @@
+package org.xmtp.android.library.frames
+
+import org.xmtp.android.library.frames.FramesConstants.OPEN_FRAMES_PROXY_URL
+
+class OpenFramesProxy(private val inner: ProxyClient = ProxyClient(OPEN_FRAMES_PROXY_URL)) {
+
+    suspend fun readMetadata(url: String): GetMetadataResponse {
+        return inner.readMetadata(url)
+    }
+
+    suspend fun post(url: String, payload: FramePostPayload): GetMetadataResponse {
+        return inner.post(url, payload)
+    }
+
+    suspend fun postRedirect(url: String, payload: FramePostPayload): FramesApiRedirectResponse {
+        return inner.postRedirect(url, payload)
+    }
+
+    fun mediaUrl(url: String): String {
+        if (url.startsWith("data:")) {
```

Comment:
I think this is fine but sometimes it can get a little funky. I like to lean on the convenience methods like 

`URI(url).scheme == "data"`